backend/plugins/mongolian_api/src/modules/ebarimt/@types/config.ts

```
@@ -0,0 +1,105 @@
+  permissionConfig?: any;
+  allowTypes: string[];
+  isCheckRemainder: boolean;
+  checkExcludeCategoryIds: string[];
+  banFractions: boolean;
+}
+
+export interface IConfigDocument extends Document, IConfig {
+  _id: string;
+}
+
+export interface IProductGroup {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)